src/main/groovy/org/pih/warehouse/inventory/CycleCountStartCommand.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.inventory
+
+import grails.validation.Validateable
+
+class CycleCountStartCommand implements Validateable {
+
+    CycleCountRequest cycleCountRequest
+
+    Integer countIndex
+
+    static constraints = {
+        countIndex(min: 0, max: 1, validator: { Integer countIndex, CycleCountStartCommand obj ->
+            CycleCount cycleCount = obj.cycleCountRequest.cycleCount
+            if (!cycleCount && countIndex != 0) {
+                return ['mustEqualZero']
```

Comment:
note that this will change when we introduce recount since starting a recount will use the same start count API. https://pihemr.atlassian.net/wiki/spaces/OB/pages/3363340321/TO+RESOLVE+page#Backend